### PR TITLE
Footer and navbar

### PR DESCRIPTION
**Pull request by lapostoj**
For the footer part, I so far added these styles in webapp and all the performance marketing apps to make sure the footer looks ok and kind of the same everywhere, so I guess it makes sense to have them here.

### CODE REVIEW

File: less/column-layout.less [hunk 1]
```diff
@@ -116,6 +116,7 @@
 .column-layout-main .container {
   // ensures we use the full width on default mac screen
   max-width: 1176px;
+  margin-left: 0 !important;
```
Comment (StevePole) [inline less/column-layout.less]: Do we need important?

File: less/transferwise.less [hunk 2]
```diff
@@ -36,6 +36,27 @@ address {
...
+	h5 {
+		a {
+			color: inherit !important;
+			font-weight: inherit;
+		}
+	}
+
+	a {
+		text-decoration: none;
+		font-weight: normal;
```
Comment (StevePole) [inline less/transferwise.less:49]: What designs is this based on?

File: less/transferwise.less [hunk 3]
```diff
@@ -36,6 +36,27 @@ address {
 }
 .footer {
     background-color: @gray-base;
+
+	.icon {
+		font-size: 50px !important;
+		margin: 0 5px 0 5px;
```
Comment (StevePole) [inline less/transferwise.less]: Not sure we need this.  We have some `.icon-lg .icon-xl .icon-xxl` utility classes.  If we do need it, I would expect it to have a total height that's divisible by 8 :-)

File: less/column-layout.less (hunk 1 above)
Comment (lapostoj) [reply less/column-layout.less]: Maybe not 🤔

File: less/transferwise.less (hunk 3 above)
Comment (lapostoj) [reply less/transferwise.less]: ah fair, will check these!

File: less/transferwise.less (hunk 2 above)
Comment (lapostoj) [reply less/transferwise.less:49]: Hmm just on not having 20 links underlined in the footer... Not sure if there are official designs for that...

Comment (StevePole) [reply less/transferwise.less:49]: We actually have some styles for 'secondary' links `a.secondary`, which we could reuse for any links inside `.footer`?

image

Comment (lapostoj) [reply less/transferwise.less:49]: Hmm didn't know those 🤔 Will have a look

Comment (lapostoj) [reply less/transferwise.less:49]: It is still underlined though 😞

**Pull request closed without merging**